app/webpacker/components/Panel/pages/CheckRecordsPage/CheckRecordsForm.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Button, Form } from 'semantic-ui-react';
+import { IdWcaSearch } from '../../../SearchWidget/WcaSearch';
+import SEARCH_MODELS from '../../../SearchWidget/SearchModel';
+import EventSelector from '../../../wca/EventSelector';
+import { events } from '../../../../lib/wca-data.js.erb';
+import { viewUrls } from '../../../../lib/requests/routes.js.erb';
+import useInputState from '../../../../lib/hooks/useInputState';
+import useCheckboxState from '../../../../lib/hooks/useCheckboxState';
+import useQueryParams from '../../../../lib/hooks/useQueryParams';
+
+const ALL_EVENTS = events.all.map((event) => event.id);
```

Comment:
There is a way to do this very, very easily and cheap with the exports that we already have. Ask me for a hint on Slack if you cannot figure it out, but believe me if I tell you the solution you're gonna hate yourself for not finding it :smile: 

(Tiny nudge: Look very closely at the two `events` exports that we already have)

Reply:
Yes! :tada: